### ISSUE TITLE
[BUG] python-fastapi generator problem with enumeration.

### ISSUE THREAD

**Issue by vlmduy** [OpenAPITools/openapi-generator#10127]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

openapi-generator-cli -g python-fastapi generator cannot generate the valid enum class.

##### openapi-generator version

5.2.0

**Comment by DarioGonzalez-Axians** [comment]
Hi! We would like to contribute and solve this issue. I think we can easily solve this and upload a PR, but we need guidance on how the generator code and the `*.mustache` templates work. Can someone explain this real quick for us?

Thanks in advance,
Axians Spain Development Team

**Comment by spacether** [comment]
Here are the docs on our mustache template library: https://github.com/spullara/mustache.java
Here are some docs on how mustache is used: http://mustache.github.io/mustache.5.html
Please read through the FastAPI generator here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonFastAPIServerCodegen.java#L130
It processes the openapi spec file in Java and renders the mustache templates into python code.
The PythonFastAPIServerCodegen templates are here:
https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources/python-fastapi

In order to contribute you would:
1. edit the templates in your fork of the repo
2. post a PR of your changes
3. run these instructions per our PR description steps:
```
./mvnw clean package 
./bin/generate-samples.sh
./bin/utils/export_docs_generators.sh
```
4. commit the updated samples
5. add any tests in the sample for fastapi here: https://github.com/OpenAPITools/openapi-generator/tree/master/samples/server/petstore/python-fastapi/tests

**Comment by liulu1998** [comment]
I encountered the same issue, has it been fixed ?

**Comment by spacether** [comment]
Nope. Enums are fully working in python-experimental though, how about trying it?

**Comment by Holt59** [comment]
@spacether `python` (prior `python-experimental`) is a client generator while `python-fastapi` is a server generator, so it's complicated to use one in place of the other.

**Comment by spacether** [comment]
Yup they are for different use cases
